tox.ini

```diff
@@ -171,3 +144,18 @@ deps =
 commands =
          python setup.py update_version
          python setup.py sdist --formats=bztar,gztar,zip bdist_wheel
+
+[testenv:pin-requirements]
+deps =
+     pip-compile-multi
+     twisted
+skip_install = True
+commands =
+         python misc/build_helpers/platform-pins.py \
+	     --remove \
+	     requirements/base.txt \
+	     requirements/platform.txt
```

Comment:
Where did platform.txt come from in the first place?

Reply:
I've added some comments to platform.txt on how it was generated. (They are a bit of a lie, as what I really did was hack pip-compile and pip to generate the hashes on linux, but I've since verified it is identical to doing what I documented).